style.css

```diff
@@ -1 +1,365 @@
 /*  Add your styling here */
+
+* {
+    font-family: 'Indie Flower', cursive;
+}
+
+body {
+    background-color: pink;
+}
+
+/* header */
+
+header {
+    border: 3px solid black;
+    margin: 5px;
+}
+
+
+.logo {
+    width: 30%;
+    border: 1px solid black;
+}
+
+/* responsive nav-bar */
+
+.menu-bar {
+    display: flex;
+    justify-content: space-around;
+    background-color: rgb(228, 112, 208);
+    padding: 20px;
+}
+
+.menu li {
+    list-style-type: none;
+    padding: 10px;
+    background-color: grey;
+}
+
+.menu {
+    position: fixed;
+    right: -100%;
+    transition: all.5s;
+
+} 
+
+ #check {
+    display: none;
+}
+
+#check:checked ~ .menu {
+    right: 0;
+}
+
+/* anchor styling */
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+a:hover {
+    font-weight: bolder;
+    color: purple;
+}
+
+/* carousel and text section */
+
+.carousel-section {
+    display:flex;
+    flex-direction: column;
+    background-color: thistle;
+    border: 5px solid black;
+    margin: 5px;
+}
+
+.hello-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+.hello-text h1 {
+    padding-top: 15px;
+    font-size: 25px;
+}
+
+.hello-text p {
+    padding: 1px 30px;
+    border-bottom: 1px solid black;
+    width: 60%;
+    padding: 25px;
+}
+
+.carousel {
+    display:flex;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    margin: 5%;
+    background-color: rgb(241, 224, 227);
+    border: 2px solid grey;
+    border-radius: 2%;
+    list-style-type: none;
+    height: 30%;
+    width: 70%;
+}
+
+.carousel > li {
+    flex-shrink: 0;
+    width: 100%;
+    height: auto;
+    margin: 10%;
+}
+
+.carousel-image {
+    width: 80%;
+    height: auto;
+    border: 3px solid black;
+    border-radius: 20%;
+    margin-top:10%;
+}
+
+
+.controls {
+    display:flex;
+    justify-content: center;
+}
+
+.control-button {
+    text-decoration: none;
+    font-size: 25px;
+    padding-left:5;
+    color: red;
+} 
+
+
+.images-controls {
+    border: 2px double black;
+    background-color: rgb(204, 154, 240);
+    width:70%;
+    height:50%;
+    margin: 13%;
+}
+
+
+/* second main section */
+
+.cakes {
+    display: flex;
+    flex-direction: column;
+    border: 3px solid black;
+    margin: 3%;
+    background-color: rgb(204, 154, 240);
+    height: 30%;
+    width: 99%;
+    overflow: hidden;
```

Comment:
consider adding padding so that the edge isn't as flush against the edge of the browser